core-java/src/test/java/com/baeldung/comparator/ComparatorUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.comparator;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import org.junit.Test;
+
+public class ComparatorUnitTest {
+    
+    @Test
+    public void whenUsingRankingComparator_thenSortedList() {
+        ArrayList<Player> footballTeam = new ArrayList<Player>();
+        Player player1 = new Player(59, "John", 20);
+        Player player2 = new Player(67, "Roger", 22);
+        Player player3 = new Player(45, "Steven", 24);
+        footballTeam.add(player1);
+        footballTeam.add(player2);
+        footballTeam.add(player3);
```

Comment:
the code for creating the collection can be in a @Before method since it's the same for both tests